core/modules/user/user.module

```diff
@@ -453,6 +453,23 @@ function user_password($length = 10) {
   return $pass;
 }
 
+/**
+ * Checks if a user role has a permission.
+ *
+ * @param string $role_name
+ *   The role machine name.
+ * @param string $permission_name
+ *   The permission to check.
+ *
+ * @return bool
+ *   TRUE if the role has the permission, FALSE if it does not have the
+ *   permission.
+ */
```

Comment:
In cases where we are introducing new functions or new parameters to existing functions, or we change the return value etc., we usually add an `@since`. Something like this:

```suggestion
 *   permission.
 *
 * @since 1.26.0 function added.
 */
```

See for example:
- https://github.com/backdrop/backdrop/blob/1.x/core/includes/image.inc#L109
- https://github.com/backdrop/backdrop/blob/1.x/core/misc/announce.js#L78
- https://github.com/backdrop/backdrop/blob/1.x/core/modules/layout/layout.api.php#L549
- https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/image.gd.inc#L32
- ...and others